osu.Game.Rulesets.Catch/Mods/CatchModFlashlight.cs

```diff
@@ -21,17 +21,11 @@ public class CatchModFlashlight : ModFlashlight<CatchHitObject>
         {
             MinValue = 0.5f,
             MaxValue = 1.5f,
-            Default = 1f,
```

Comment:
You forgot to add (1f) to BindableFloat

Reply:
I don't think it makes a difference though does it?

Reply:
You are right , thanks.

Reply:
Fixed.